convert2rhel/logger.py

```diff
@@ -113,15 +115,18 @@ def archive_old_logger_files(log_name, log_dir):
 
     stat = os.stat(current_log_file)
 
-    # Get the last time the file was modified
-    last_modified_at = stat.st_mtime
+    # Get the last modified time in UTC
+    last_modified_at = gmtime(stat.st_mtime)
+
+    # Format time to a human-readable format
+    formatted_time = strftime("%Y%m%dT%H%M%SZ", last_modified_at)
```

Comment:
@bocekm you think we should include the extra milliseconds in the format string too? 
```suggestion
    formatted_time = strftime("%Y%m%dT%H%M%S%fZ", last_modified_at)
```

Reply:
No need for ms there :)